crates/pop-common/src/sourcing/mod.rs

```diff
@@ -160,7 +161,8 @@ impl Source {
 }
 
 /// A binary sourced from GitHub.
-#[derive(Clone, Debug, PartialEq)]
+#[derive(Clone, Debug, Derivative)]
```

Comment:
I was initially confused about why we needed to introduce the new `derivative` dependency, but I see now that without it we’d have to manually implement `PartialEq`. Just confirming that’s the reason for adding it, right?